src/App.js

```diff
@@ -1,26 +1,53 @@
 import React from 'react';
 import logo from './logo.svg';
 import './App.css';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div className="links">
+        <nav>
+          <ul className="linkslist">
+            <li>
+              <NavLink activeClassName="selected" to="/list">
+                List
+              </NavLink>
+            </li>
+            <li>
+              <NavLink activeClassName="selected" to="/add">
+                Add an Item
+              </NavLink>
+            </li>
+          </ul>
+        </nav>
+
+        {/* A <Switch> looks through its children <Route>s and
+            renders the first one that matches the current URL. */}
+        <Switch>
+          <Route path="/list">
+            <List />
+          </Route>
+          <Route path="/add">
+            <Add />
+          </Route>
+        </Switch>
+      </div>
+    </Router>
   );
 }
 
+function List() {
+  return <div></div>;
```

Comment:
Just so we can see something on the screen, I would add something in the div. Even something as small as List works.